daisy_workflows/image_import/import_image.sh

```diff
@@ -43,6 +43,27 @@ echo "DISKNAME: ${DISKNAME}" 2> /dev/null
 echo "ME: ${ME}" 2> /dev/null
 echo "ZONE: ${ZONE}" 2> /dev/null
 
+function waitUntilDeviceAttached() {
+  local devPath="${1}"
+  local capacityGb="${2}"
+
+  echo "Import: Checking for $devPath ${capacityGb}G"
+  for t in {1..60}; do
+    if [[ -e ${devPath} ]]; then
+      capacity=$(lsblk ${devPath})
+      if [[ "${capacity}" =~ "${capacityGb}G" ]]; then
```

Comment:
Why is capacity check needed? Did you see cases where the device is attached but size is incorrect, like 0?

Reply:
Yeah, I've seen examples where the disk image requires about 30GB, we resize it, and qemu-img fails at about 10GB (which is the normal disk size).